T/TheCannon/Deps.toml

```diff
@@ -3,3 +3,6 @@ ForwardDiff = "f6369f11-7733-5829-9624-2563aa707210"
 LinearAlgebra = "37e2e46d-f89d-539d-b4ee-838fcccc9c8e"
 Optim = "429524aa-4258-5aef-a3af-852621145aeb"
 Statistics = "10745b16-79ce-11e8-11f9-7d13ad32a3b2"
+
+["0.2-0"]
+Documenter = "e30172f5-a6a5-5a46-863b-614d45cd2de4"
```

Comment:
I don't think you need Documenter as a dependency.

Reply:
This should be a dependency for the package (not sure if this is the right terminology) in `/docs`, correct?

Reply:
Right, and you already have it there: https://github.com/ajwheeler/TheCannon.jl/blob/71ed0b16508858d7f98c90eab94059362cadf386/docs/Project.toml#L2 but since you are not using `Documenter` from inside `TheCannon` you don't need to include it in the main `Project.toml`.